packages/jest-matchers/src/matchers.js

```diff
@@ -139,16 +139,35 @@ const matchers: MatchersObject = {
         `'${expected}' (using <=)`;
     return {message, pass};
   },
+
+  toContain(actual: number | string, expected: any) {
```

Comment:
shall we add `toInclude`, given that ES2015 now has `(Array|String).prototype.includes`?